.github/workflows/linting.yml

```diff
@@ -37,6 +37,7 @@ jobs:
         uses: actions/checkout@v3
         with:
           repository: pmagentur/linting-javascript
+          ref: master
```

Comment:
I guess this is not correct or @pmMarcel ? Or if is is correct we should change it in all lintings repos.

Reply:
This is actually correct, though this PR should be normally only to development first ^^

Reply:
Changed the base branch for the PR